buffer.go

```diff
@@ -0,0 +1,17 @@
+package run
+
+import (
+	"github.com/djherbis/buffer"
+)
+
+// maxBufferSize denotes the maximum size of each buffer. Overflows are written to disk
+// at increments of this size.
+var maxBufferSize int64 = 128 * 1024
+
+// bufferPool will never return an error.
+//
+// Uses unbounded buffers that create files of size fileBuffersSize to store overflow.
+func makeBuffer() buffer.Buffer {
```

Comment:
```suggestion
func makeUnboundedBuffer() buffer.Buffer {
```

Could we rename this func to ensure we don't get puzzled when reading the code? With all the other buffer uses, it's not obvious why we would use ours here by just reading the function name.